src/models/LowerInsert.js

```diff
@@ -0,0 +1,18 @@
+const LowerInsert = 
+    `CREATE OR REPLACE FUNCTION LowerInsert() RETURNS trigger AS
+    $$
+    DECLARE
+        col TEXT := TG_ARGV[0];
+        target TEXT;
+    BEGIN
+        EXECUTE format('SELECT ($1).%I', col)
+        USING NEW
+        INTO target;
+        target = LOWER(target);
+        NEW := NEW #= hstore(col, target);
+        RETURN NEW;
+    END;
+    $$ LANGUAGE plpgsql;
+    `
+
+module.exports = LowerInsert;
```

Comment:
I've heard in the past that it's best to avoid triggers when possible. For our purposes, as long as everything works, I see no problem. No need to change anything as long as everything works.